opendbc/car/torque_data/override.toml

```diff
@@ -65,6 +65,7 @@ legend = ["LAT_ACCEL_FACTOR", "MAX_LAT_ACCEL_MEASURED", "FRICTION"]
 "HYUNDAI_CUSTIN_1ST_GEN" = [2.5, 2.5, 0.1]
 "LEXUS_GS_F" = [2.5, 2.5, 0.08]
 "HYUNDAI_STARIA_4TH_GEN" = [1.8, 2.0, 0.15]
+"GENESIS_GV70_ELECTRIFIED_1ST_GEN" = [2.5, 2.5, 0.1]
```

Comment:
Adjusting lateral accel control parameters to match online learned performance.

```suggestion
"GENESIS_GV70_ELECTRIFIED_1ST_GEN" = [1.9, 1.9, 0.09]
```

![image](https://github.com/user-attachments/assets/5e9ae85f-02e8-46f9-9cdd-b59a793c493f)